src/js/modules/articles/components/ArticleList.js

```diff
@@ -51,15 +51,18 @@ const ArticleList = ({ classes, articles, title, label }) => {
   }
   return (
     <Grid className={classes.ArticleList}>
-      {title && (
-        title === "VOICES" ? (
+      {title &&
+        (title === "VOICES" ? (
```

Comment:
Ehh. That's interesting. Hard coded?

Reply:
Yea. There's only one subsection with an art banner right now.